src/olympia/users/management/commands/createsuperuser.py

```diff
@@ -80,6 +90,23 @@ def handle(self, *args, **options):
                 'api-secret': apikey.secret
             }))
 
+        if options.get('ui_test', False):
+            # json object for variables file
+            variables = {}
+            variables['api'] = {}
+
+            # export to variables.json object
+            variables['api'].update({
+                'olympia.dev': dict([
+                    ('username', user.username,),
+                    ('jwt_issuer', apikey.key),
+                    ('jwt_secret', apikey.secret)
+                ])
+            })
```

Comment:
I believe all that can be written as

```python
credentials = {
    'api': {
        'olympia.dev': {
            'username': user.username,
            'jwt_issuer': apikey.key,
            'jwt_secret': apikey.secret,
        }
}
```

Also, why do you have `olympia.dev` in there? If you need this, it probably should be a setting of some sorts.